stellar/its.js

```diff
@@ -187,6 +201,13 @@ if (require.main === module) {
             );
         });
 
+    program
+        .command('execute <sourceChain> <messageId> <sourceAddress> <payload>')
```

Comment:
Not technically needed since the gateway execute is sufficient, just need to provide ITS as the destination address. But we can include it for convenience in this specific case